xml/System.Web/SameSiteMode.xml

```diff
@@ -14,7 +14,20 @@
   </Base>
   <Docs>
     <summary>Specifies constants that indicate the value for the SameSite attribute of the cookie.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
```

Comment:
Why are the remarks in HttpCookie out of sync with these?

Reply:
@Tratcher That was intentional. On the `HttpCookie` page, I wanted to highlight the changes to the default value of that property. On the `SameSiteMode` page I wanted to highlight the changes in the enum fields. I was trying to highlight what I thought would be most important for readers that landed on each page. 

What would you change?

Reply:
Good point. I guess the main thing missing from the HttpCookie doc is the https://tools.ietf.org/html/draft-west-cookie-incrementalism-00 link.